src/clr-angular/utils/_components.clarity.scss

```diff
@@ -44,9 +44,6 @@
 @import '../typography/typography.clarity';
 @import '../typography/code.clarity';
 
-// A11Y
-@import '../utils/a11y.clarity';
```

Comment:
Don't we need this?

Reply:
Duplicate import, was imported already at the top of the file.